scripts/ci/junit2slack/main.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
```

Comment:
the imports are out of order, which I think `style-checks` caught. You can run `make style` or `quickstyle`, if you have the https://github.com/stackrox/workflow scripts